### DISCUSSION TITLE
Far future idea - Revolt

### DISCUSSION THREAD

**Discussion by 1ByteBit** [thatguycy/WorldDynamics-Engine#20] [Ideas]
Revolts, revolutions, coup d'état, etc. Basically a system to allow a forced change in ownership within countries/nations/etc

Definitely a far off feature however

**Comment by stoleyourharvs** [comment]
These ideas might not be as far off as you think 👀

**Reply by 1ByteBit** [reply]
Oh my

**Reply by PhantomOfficial1112** [reply]
bet
